### PR TITLE
Add staging augmenter to the release.groovy

### PR DESCRIPTION
**Pull request by xstefank**
Is there a way to verify the script before the merge?

### CODE REVIEW

**Comment by radcortez** [comment]
> Is there a way to verify the script before the merge?

I don't think so :(

File: buildScripts/release.groovy
Comment (radcortez) [inline buildScripts/release.groovy:44]: Some systems don't have `wget` by default. I guess we need to try it to see if it work.

**Review by radcortez — APPROVED** [review]

**Comment by xstefank** [comment]
if this will be trial and error, I guess we can proceed and merge?

**Comment by xstefank** [comment]
nevermind, since config is going to be released any time now I will wait for phase 2 to be staged before tinkering with the release job.

File: buildScripts/release.groovy
Comment (Emily-Jiang) [reply buildScripts/release.groovy:44]: @xstefank can you move the files in your github repo to this repo?

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]

**Comment by Emily-Jiang** [comment]
@xstefank it is a good idea to let Rest Client and JWT to release this week before merging this and next week we can put in all of the remaining specs for release.

File: buildScripts/release.groovy
Comment (xstefank) [reply buildScripts/release.groovy:44]: I would like to first verify that the build will pass. The binary is Linux 64 compatible. Maybe it will need to adjust for the Jenkins environment. But of course, after the job passes, I would like to push everything here.

**Comment by xstefank** [comment]
OK, I will hold this merge until all specifications from phase 2 are staged.

**Pull request merged**
